apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -720,7 +720,11 @@ export const PublishButton = ({ projectId }: PublishProps) => {
   return (
     <FloatingPanelPopover modal open={isOpen} onOpenChange={handleOpenChange}>
       <FloatingPanelAnchor>
-        <Tooltip side="bottom" content={tooltipContent}>
+        <Tooltip
+          side="bottom"
+          content={tooltipContent ?? "Publish to Webstudio Cloud"}
+          sideOffset={parseFloat(rawTheme.spacing[5])}
```

Comment:
Number.parseFloat

Reply:
I will never memorize this, please add a linter or just ignore